.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Discord
+    about: Ask questions on our Discord Server.
+    url: https://discord.gg/EUHuJHt
+  - name: Documentation
+    about: Useful documentation about Modrinth, its API and how you can contribute.
```

Comment:
Needs an oxford comma

Reply:
Here's two examples:
  + "foo, bar and baz"
  + "foo, bar, and baz"
The extra comma in the second is an Oxford comma.